Parser.py

```diff
@@ -0,0 +1,98 @@
+from Mailer import Mailer
+import urllib.request
+
+mailer = Mailer()
+
+class Parser:
+    # Load data from enviromental variables, set the parse URL and load all the parsed URLs
+    def __init__(self, parseURL):
+        print('Initializing parser')
+        self.ads = []
+        file = open('url_list.txt', 'r')
+        lines = file.readlines()
```

Comment:
You don't need this if you let your for loop do the reading.
Instead of reading lines first you can do this:
```suggestion
        for line in file:
```